data/core/macros/amla.cfg

```diff
@@ -12,19 +31,283 @@
         [effect]
             apply_to=hitpoints
             increase_total=3
-            heal_full=yes
-        [/effect]
-        [effect]
-            apply_to=max_experience
-            increase=20%
-        [/effect]
-        [effect]
-            apply_to=status
-            remove=poisoned
-        [/effect]
-        [effect]
-            apply_to=status
-            remove=slowed
         [/effect]
+        {AMLA_EFFECTS_HEAL_AND_RAISE_XP}
     [/advancement]
 #enddef
+
+#
+# This is a resource intended for use in select mainline campaigns, and should not be relied upon for UMC.
+# It may change in ways that aren't backwards-compatible with or that negatively affect the balance of UMC content that uses it.
+#
+# Include this resource in your [campaign] tag to replace the default +3hp AMLA with more powerful AMLA choices.
+# All max-level units gain a +8hp AMLA option, and units with melee/ranged attacks gain options for increased melee/ranged damage.
+# Units whose melee attacks all have 1 strike gain a +4 melee dmg option. Units whose melee attacks all have <=2 strikes gain a +2 melee dmg option. Otherwise, units gain a +1 melee dmg option.
+# The same pattern follows for ranged attacks.
+#
+# The optional `use_stronger_amlas_filter` variable can be used to modify which units this macro affects. `use_stronger_amlas_filter` defaults to `side=1`
+[resource]
+    id=use_stronger_amlas
+
+    #############################
+    # INITIALIZE AMLA FILTER
+    #############################
+    [event]
+        name=prestart
```

Comment:
Might be nice to set this event with lower priority.

Reply:
I think that setting a customer filter should work fine regardless of the order in which things happen. If I'm not mistaken:

If I set a custom filter first, this event won't trigger because the variable won't be null. If I set a custom filter after, I override the default setting from th eevent.

Reply:
Fair enough, consumers can use start event to change filter.

Reply:
this deends of course on _how_ they set it, if they for example set it the same way as this even does (by setting `use_stronger_amlas_filter.side`), it won't overwrite it.